app-backend/backsplash-server/Dockerfile

```diff
@@ -1,15 +1,7 @@
-FROM daunnc/openjdk-gdal:2.3.2
+FROM quay.io/azavea/openjdk-gdal:2.3.2-jdk8-slim
 
 ENV ASPECTJ_WEAVER_VERSION 1.8.10
 
-RUN \
```

Comment:
I don't see why we need this, because we already have an `sbt` launcher script checked into the repository.